projects/libfdk-aac/aacDecoder_Open.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "aacdecoder_lib.h"
+#include <stdint.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  HANDLE_AACDECODER aacDecoderInfo = NULL;
+
+  TRANSPORT_TYPE transportType;
+  INT nrOfLayers;
+
+  if (Size != 8) return 0;
+
+  transportType = (TRANSPORT_TYPE)(Data[0] + (Data[1] << 8) + (Data[2] << 16) +
+                                   (Data[3] << 24));
+  nrOfLayers =
+      (UINT)(Data[4] + (Data[5] << 8) + (Data[6] << 16) + (Data[7] << 24));
+
+  aacDecoderInfo = aacDecoder_Open(transportType, nrOfLayers);
```

Comment:
This seems like a fairly shallow target. I suppose opening with malicious (transportType, nrOfLayers) pairs have been buggy in the past?